src/components/quotes.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from 'react';
+import './quotes.css';
+
+const Quotes = () => {
+  const [data, setData] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(false);
+
+  const apiKey = process.env.REACT_APP_API_KEY;
+  const url = 'https://api.api-ninjas.com/v1/quotes?category=';
+  const category = 'love';
+
```

Comment:
- [ ] Hello there, @lincoln1883. So far, you've done an excellent job fetching the quotes from the Ninja API. However, it appears that we are not receiving any quotes from the API. Please make sure to correct this. 
I believe you should double-check the API key configuration. Because I don't see where you set up your environment variable (REACT_APP_API_KEY) in your project.

### Demo :point_down: 

[Screencast from 31.05.2023 18:37:17.webm](https://github.com/lincoln1883/math-magicians/assets/89228743/c834c8bb-a57b-4d28-b183-69c4057c96a8)

Reply:
Thank you for your feedback i have a .env file containing the api key which is refered to in the git ignore file.